lib/wai-bindgen-wasmer/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "wai-bindgen-wasmer"
```

Comment:
Shouldn't this live in `wai`? (I'm ok for this living here, but want to clarify just in case!)

Reply:
We talked about this quite a bit with @Michael-F-Bryan .

The problem is that we now actually use the WAI bindings for WASI.
That means we need to be able to make changes to the bindgen crate and point it towards the latest version of Wasmer.
Thats not possible if the crate is not in the Wasmer repo.

Reply:
Going to resolve since it should be here, but feel free to re-open.

Reply:
SGTM